Docs: Update schedules.md for deployments from cli and python file (#6694)
PrefectHQ/prefect: docs/concepts/schedules.md

You can add a schedule to any flow [deployment](/concepts/deployments/). The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.

> There are four recommended possible ways to create a schedule for a deployment:

Reviewer: ```suggestion
There are four recommended ways to create a schedule for a deployment:
```

> There are four recommended possible ways to create a schedule for a deployment:

Reviewer: ```suggestion
There are four recommended ways to create a schedule for a deployment:
```

- Use the Prefect UI

> - Use the `--schedule` flag with the CLI `deployment build` command

Author: ```suggestion
- "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command
```

> - Use the `--schedule` flag with the CLI `deployment build` command

Reviewer: Unfortunately we don't support a single `--schedule` flag, there are 3 separate flags, one for each schedule type; not sure of the best way to say that here.

Author: Good catch.  I'll change to "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command.

> - "Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command

Reviewer: ```suggestion
- Use the `cron`, `interval`, or `rrule` flags with the CLI  `deployment build` command
```
accidental double quote

- Use the `schedule` parameter with a Python deployment file

> - Edit the deployment YAML file's `schedule` section

Reviewer: ```suggestion
- Manually edit the deployment YAML file's `schedule` section
```

- [`RRule`](#rrules) is best suited for deployments that rely on calendar logic for simple recurring schedules, irregular intervals, exclusions, or day-of-month adjustments.

## Creating schedules through the CLI with the `deployment build` command

When you build a deployment from the CLI you can use `cron`, `interval`, or `rrule` flags to set a schedule. 

<div class="terminal">
```bash
prefect deployment build demo.py:pipeline -n etl --cron "0 0 * * *"
```
</div>

> This schedule will set flow runs in a deployment every day at midnight. You cannot set a timezone for your schedule from the CLI at the present time.

Reviewer: ```suggestion
This schedule will create flow runs for this deployment every day at midnight. You cannot set a timezone for your schedule from the CLI at the present time.
```
